requirements.txt

```diff
@@ -30,3 +30,4 @@ fann2==1.0.7
 padaos==0.1.9
 precise-runner==0.2.1
 petact==0.1.2
+pyxdg>=0.26
```

Comment:
Let's keep the requirement locked pyxdg==0.26 (using the loose requirements thing)

Reply:
Good call, fixed.